README.md

```diff
@@ -44,6 +44,11 @@ Requires a [Go installation](https://go.dev/dl/) to run the examples.
     go run hello.go
     ```
 
+4. *Option* instead of step 3. Use [air](https://github.com/cosmtrek/air) to auto reload your code after you save your code. Then you will see your code result without any action. You can simplely focus on your code.
```

Comment:
I'd change this line to not be a step, instead be a paragraph below the steps. Along with a grammar fix this results in:
```markdown
Optionally, instead of step 3, use [air](https://github.com/cosmtrek/air) to auto reload your code after you save your code.
Then you will see your code output without any action. You can simply focus on your code.
```

Reply:
changed and pr again.